content/english/Kusto-KC7/activity-4.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Activity 4: Kusto Query Language 101"
+description: ""
+draft: false
+weight: 12
+---
+{{< youtube VH8NAJjlPa4 >}}
+
+
+{{< notice tip >}} This is where we want you to pull up the Scoreboard and ADX viewer from earlier in seperate windows to complete the remaining exercises. If you can't or don't want to use the Scoreboard, all questions for this section are listed below.  
```

Comment:
```suggestion
{{< notice tip >}} This is where we want you to pull up the Scoreboard and ADX viewer from earlier in separate windows to complete the remaining exercises. If you can't or don't want to use the Scoreboard, all questions for this section are listed below.  
```

Reply:
It was kind of hard to tell what you wanted here, looked like you wanted me to make a few specific words stand out, Looks odd to me but we can discuss.

Reply:
Ok i realized github must be highlighting if, or, and words on default. This was just a spelling check for "separate". Updated.